pkg/utils/key.go

```diff
@@ -0,0 +1,70 @@
+package utils
+
+import (
+	"bytes"
+	"encoding/hex"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/pingcap/errors"
+	"github.com/spf13/pflag"
+)
+
+// ParseKey parse key by given format
+func ParseKey(flags *pflag.FlagSet, key string) (string, error) {
+	switch flags.Lookup("format").Value.String() {
+	case "raw":
+		return key, nil
+	case "encode":
+		return decodeKey(key)
```

Comment:
I think this format is better to be called "escaped", and the `decodeKey` is actually `unescapeKey`.

Reply:
Sure, I found this function in PD, but  seems escaped is meaningful